mantidproject/mantid
# Fix registering an algorithm duplicating the list of algorithms in the algorithm selector widget

**Description of work.**
What the title says.

**To test:**
Copy the script from the associated issue and run it in Workbench. Check that the list of algorithms in the algorithm selector widget is not repeated. Type the name of an algorithm into the search box and check that it isn't listed more than once.

Fixes #27087 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
